src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -46,6 +50,26 @@ export default class SaveInProgressIntro extends React.Component {
             <br/>
           </div>
         );
+      } else if (verifyRequiredPrefill) {
+        alert = (
+          <div>
+            <div className="usa-alert usa-alert-info schemaform-sip-alert">
+              <div className="usa-alert-body">
+                <strong>Note:</strong> Since you’re signed in to your account, you can save your form in progress, and come back later to finish filling it out.
+                <ul>
+                  <li>You have 1 year from the date you start or update your application to submit the form. After 1 year, the form won’t be saved, and you’ll need to start over.</li>
```

Comment:
Can we pull this alert out and pass it in as another prop since this content is specific to 526?